src/cloud-cli/applications/get-app-logs.ts

```diff
@@ -1,12 +1,18 @@
 import axios from "axios";
 import { GlobalLogger } from "../../telemetry/logs";
 import { getCloudCredentials } from "../utils";
+import path from "node:path";
 
-export async function getAppLogs(appName: string, host: string): Promise<number> {
+export async function getAppLogs(host: string): Promise<number> {
```

Comment:
At some point, we need to rewrite this to use the new logging export/observability systems

Reply:
Roger, I'll make a note about this in my doc